Expand WebAssembly docs (#71)
denoland/manual: webassembly.md

## Using WebAssembly

> Designed to be used alongside JavaScript to speed up key application components,
> [WebAssembly](https://webassembly.org/) has a much higher execution speed than
> JavaScript, similar to C and C++. Deno can execute WebAssembly modules with the
> same interfaces that
> [browsers provide](https://developer.mozilla.org/en-US/docs/WebAssembly).

Reviewer: ```suggestion
Designed to be used alongside JavaScript to speed up key application components,
[WebAssembly](https://webassembly.org/) can have much higher, and more consistent execution speed than
JavaScript, similar to C, C++, or Rust. Deno can execute WebAssembly modules with the
same interfaces that
[browsers provide](https://developer.mozilla.org/en-US/docs/WebAssembly).
```

In this chapter we will discuss:

1. [Using WebAssembly in Deno](webassembly/using_wasm.md)

> 2. [Using the Asynchronous WebAssembly APIs](webassembly/using_wasm_async.md)

Reviewer: ```suggestion
2. [Using the asynchronous WebAssembly APIs](webassembly/using_wasm_async.md)
```
